Source/Extensions/Blazorise.DataGrid/_DataGridPagination.razor

```diff
@@ -50,53 +63,89 @@
+    {
+        @if ( ShowColumnChooser() )
         {
             <Field Display="Blazorise.Display.InlineBlock">
-                @{
-                    var newButtonString = Localizer.Localize( ParentDataGrid.Localizers?.NewButtonLocalizer, "New" );
-                    var editButtonString = Localizer.Localize( ParentDataGrid.Localizers?.EditButtonLocalizer, "Edit" );
-                    var deleteButtonString = Localizer.Localize( ParentDataGrid.Localizers?.DeleteButtonLocalizer, "Delete" );
-                    var clearFilterButtonString = Localizer.Localize( ParentDataGrid.Localizers?.ClearFilterButtonLocalizer, "Clear Filter" );
-                    var saveBatchButtonString = Localizer.Localize( ParentDataGrid.Localizers?.SaveBatchButtonLocalizer, "Save Changes" );
-                    var cancelBatchButtonString = Localizer.Localize( ParentDataGrid.Localizers?.CancelBatchButtonLocalizer, "Cancel Changes" );
-
-                    @ButtonRowTemplate( new()
-                    {
-                        NewCommand = new()
-                        {
-                            Clicked = NewClick,
-                            LocalizationString = newButtonString,
-                        },
-                        EditCommand = new()
-                        {
-                            Clicked = EditClick,
-                            LocalizationString = editButtonString,
-                            Item = SelectedRow
-                        },
-                        DeleteCommand = new()
-                        {
-                            Clicked = DeleteClick,
-                            LocalizationString = deleteButtonString,
-                            Item = SelectedRow
-                        },
-                        ClearFilterCommand = new()
+                <Dropdown @bind-Visible="@dropdownColumnChooserVisible">
+                    <DropdownToggle @onclick:stopPropagation>
+                        <Icon Name="@IconName.Table" Margin="Blazorise.Margin.Is1.FromStart" Clicked="() => {}" /> Columns
+                    </DropdownToggle>
+                    <DropdownMenu MaxMenuHeight="200px">
+                        @if ( ParentDataGrid.ColumnChooserTemplate is null ) 
                         {
-                            Clicked = ClearFilterClick,
-                            LocalizationString = clearFilterButtonString,
-                        },
-                        SaveBatchCommand = new ()
-                        {
-                            Clicked = SaveBatchClick,
-                            LocalizationString = saveBatchButtonString,
-                        },
-                        CancelBatchCommand = new()
+                            @foreach ( var column in ParentDataGrid.GetColumns().Where( x => x.IsRegularColumn ) )
+                            {
+                                <Row Margin="Blazorise.Margin.Is0">

```

Comment:
We should use the Dropdown built-in checkbox feature instead of adding the Check box here. That way the UI will be more consistent across all providers.

Reply:
I think I had a limitation, but I'll try again, I can't remember.

Reply:
Works perfectly. ;Maybe I was just dumb...